airbyte/_util/api_util.py

```diff
@@ -1564,6 +1564,51 @@ def get_connector_builder_project_for_definition_id(
     return json_result.get("builderProjectId")
 
 
+def update_connector_builder_project_testing_values(
+    *,
+    workspace_id: str,
+    builder_project_id: str,
+    testing_values: dict[str, Any],
+    spec: dict[str, Any],
```

Comment:
I'm not confident with the spec implementation and I didn't know that we needed it. I think you need to do more testing to see (1) do we need this, (2) do we need to create it ourselves, (3) is there a reliable way to auto-generate it if yes to 1 and 2.